### DISCUSSION TITLE
How to release?

### DISCUSSION THREAD

**Discussion by rakuri255** [rakuri255/UltraSinger#84] [Q&A]
I've been trying to create a release for a while now, but honestly don't know how to release a python program. 

Pyinstaller has its problems, see CI branch. And if you add cuda library you quickly get to 5 Gb for the application.

Possibly wheel would be an option, I have not yet tried. 

Does anyone have experience and could help out here?

**Comment by eadmaster** [comment]
i suggest using docker: https://github.com/rakuri255/UltraSinger/discussions/103

**Reply by rakuri255** [reply]
Good idea. Will look into it!
